src/containers/CodesplainAppBar.jsx

```diff
@@ -7,6 +7,7 @@ import {
 } from 'material-ui';
 import { withRouter } from 'react-router';
 import cookie from 'react-cookie';
+import _ from 'lodash';
```

Comment:
I'm not saying you cannot use lodash, but rather that importing the default lodash object isn't allowed. Import only the functions you need.